### PR TITLE
Little tidy up password fragments

### PR DESCRIPTION
**Pull request by feedmypixel**
- abstracting some common functionality into fragments
- rename of passwordDetail fragment and fragment references

### CODE REVIEW

File: frontend/app/views/tier/upgrade/paidToPaid.scala.html [hunk 1]
```diff
@@ -144,24 +144,8 @@ <h2 class="page-section__headline">
                 <form action="@routes.TierController.upgrade(targetTier)" method="POST" class="js-form" id="payment-form">
                     @CSRF.formField
                     <div class="fieldset__fields js-password-container">
```
Comment (davidrapson) [inline frontend/app/views/tier/upgrade/paidToPaid.scala.html]: Is there a missing `.fieldset` parent class here?

File: frontend/app/views/fragments/form/forgottenPassword.scala.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@(text: String)
+
+@import configuration.Config
+
+<a href="@Config.idWebAppUrl/reset">@text</a>
```
Comment (davidrapson) [inline frontend/app/views/fragments/form/forgottenPassword.scala.html]: FRAGMENTS!

**Comment by davidrapson** [comment]
:+1: minor BEM fragment comment but otherwise looks good.

File: frontend/app/views/tier/upgrade/paidToPaid.scala.html (hunk 1 above)
Comment (feedmypixel) [inline frontend/app/views/tier/upgrade/paidToPaid.scala.html]: yep looks like it - I will fix up in another PR

**Pull request merged**